config/routes.rb

```diff
@@ -6,6 +6,6 @@
     get "/login", to: "sessions#new"
     post "/login", to: "sessions#create"
     delete "/logout", to: "sessions#destroy"
-    resources :users, only: %i(create show)
+    resources :users, except: %i(new)
```

Comment:
Chỉ 1 action thì k cần dùng dạng array đâu
Thế này đc rồi
```suggestion
    resources :users, except: :new
```

Reply:
oke